config/initializers/carrierwave.rb

```diff
@@ -1,12 +1,17 @@
-if ENV["S3_ACCESS_KEY_ID"]
-  CarrierWave.configure do |config|
+CarrierWave.configure do |config|
+  if ENV["S3_ACCESS_KEY_ID"]
+    config.storage :fog
+
     config.fog_credentials = {
       provider:              "AWS",
       aws_access_key_id:     ENV["S3_ACCESS_KEY_ID"],
       aws_secret_access_key: ENV["S3_SECRET_ACCESS_KEY"],
-      region:                ENV["S3_REGION"]
+      region:                ENV["S3_REGION"],
+      path:                  (ENV.has_key?("USE_SHARED_S3_BUCKET") ? ENV["HEROKU_APP_NAME"] : nil)
```

Comment:
__[rubocop]__
* Style/DeprecatedHashMethods: `Hash#has_key?` is deprecated in favor of `Hash#key?`. [:ref](https://github.com/bbatsov/ruby-style-guide#hash-key)